subgraphs/aave-forks/src/helpers.ts

```diff
@@ -1015,9 +1015,7 @@ export function getOrCreateMarket(
     market.cumulativeBorrowUSD = BIGDECIMAL_ZERO;
     market.cumulativeLiquidateUSD = BIGDECIMAL_ZERO;
     market.inputTokenBalance = BIGINT_ZERO;
-    market.inputTokenPriceUSD = BIGDECIMAL_ZERO;
```

Comment:
let's update the `graph-cli/ts` versions to the latest. I don't think you can not set required vars in the newer versions.

Reply:
I am using graph-cli 0.37.7 and it works fine for me.